.github/workflows/ruff.yml

```diff
@@ -0,0 +1,12 @@
+name: Ruff
+on: [ push, pull_request ]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: chartboost/ruff-action@v1
```

Comment:
this step already runs ruff, by default the `check` command

Reply:
We need two ruff steps: 
* lint - `ruff check`
* check formatting - `ruff format --check`

Currently there is only one

Reply:
Should I just add onother
`- uses: chartboost/ruff-action@v1`?

Reply:
yes